.github/workflows/build_monolithic.yml

```diff
@@ -306,7 +306,7 @@ jobs:
           popd
 
       - name: Upload Wine
-        uses: actions/upload-artifact@v2
+        uses: ./github/actions/upload-artifact@v2
```

Comment:
This workaround lightly won’t be needed for published actions only for the custom ones provided by this repository (I’d think)

Reply:
You are perfectly right - this is the result of a stupid mass-replace of mine...

Reply:
Oh I’ve done this multiple times with winebuild